From 665264d938a8631167945279a364f171a17ac74a Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Wed, 7 Jan 2015 12:25:34 -0800
Subject: [PATCH 051/184] netdev: octeon3-ethernet: Cleanup use of boolean
 variables.

Source: Cavium Networks, Inc.
MR: 00000
Type: Integration
Disposition: Merged from Octeon Tree
ChangeID: 3b32bbd0b1158c5ca5130e4c4192f1e322c12f02
Description:

Signed-off-by: David Daney <david.daney@cavium.com>
Signed-off-by: Corey Minyard <cminyard@mvista.com>
[Original patch taken from octeon-linux-kernel-patches-SDK-3.1.2-release]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/net/ethernet/octeon/octeon3-ethernet.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/octeon/octeon3-ethernet.c b/drivers/net/ethernet/octeon/octeon3-ethernet.c
index ad18b48..c637486 100644
--- a/drivers/net/ethernet/octeon/octeon3-ethernet.c
+++ b/drivers/net/ethernet/octeon/octeon3-ethernet.c
@@ -1802,7 +1802,7 @@ static int octeon3_eth_ndo_start_xmit(struct sk_buff *skb, struct net_device *ne
 	}
 
 	/* SSO can only fall behind when the skb is not recycled */
-	if (can_recycle_skb == false) {
+	if (!can_recycle_skb) {
 		oen = octeon3_eth_node + priv->numa_node;
 		aq_cnt.u64 = cvmx_read_csr_node(oen->numa_node,
 				CVMX_SSO_GRPX_AQ_CNT(oen->tx_complete_grp));
@@ -1810,8 +1810,8 @@ static int octeon3_eth_ndo_start_xmit(struct sk_buff *skb, struct net_device *ne
 
 	/* Drop the packet if pko or sso are not keeping up */
 	backlog = atomic64_inc_return(&priv->tx_backlog);
-	if (unlikely(backlog > MAX_TX_QUEUE_DEPTH) ||
-	    (can_recycle_skb == false && aq_cnt.s.aq_cnt > 100000)) {
+	if (unlikely(backlog > MAX_TX_QUEUE_DEPTH ||
+		     (!can_recycle_skb && aq_cnt.s.aq_cnt > 100000))) {
 		if (use_tx_queues) {
 			netif_stop_queue(netdev);
 		} else {
-- 
1.9.1

